src/Model.jl

```diff
@@ -103,8 +103,8 @@ function Model(lattice::AbstractMatrix{T};
         norm(lattice[:, i]) == norm(lattice[i, :]) == 0 || error(
             "For 1D and 2D systems, the non-empty dimensions must come first")
     end
-    cond(lattice[1:n_dim, 1:n_dim]) > 1e-5 || (
-        @warn "Your lattice is badly conditioned, the computation is likely to fail.")
+    _check_well_conditioned(lattice[1:d, 1:d]) || @warn (
```

Comment:
This should be `1:n_dim` now.

Reply:
ah yes!

Reply:
https://github.com/JuliaMolSim/DFTK.jl/pull/476/commits/cb466c1903f805af1008fb009472712cef51f8ea